pkg/apis/kops/instancegroup.go

```diff
@@ -89,6 +90,9 @@ type InstanceGroupSpec struct {
 
 	// NodeLabels indicates the kubernetes labels for nodes in this group
 	NodeLabels map[string]string `json:"nodeLabels,omitempty"`
+
+	// Taints indicates the kubernetes taints for nodes in this group
+	Taints []string `json:"taints,omitempty"`
```

Comment:
Yeah, I was on the fence and defaulted to the simple option. But a stronger type would help with early validation. It sounds like this PR is going to sit until the relevant bits of 1.6 are settled. So I'll wait to see what makes sense at that point.